### DISCUSSION TITLE
请问这个模型可以拿到ocr坐标吗

### DISCUSSION THREAD

**Discussion by farierer** [RapidAI/RapidTable#34] [Q&A]
您好，我看这个模型的是使用ocr做的，我知道这个模型是为了还原markdown格式，但是我还是想请问是否可以拿到每个单元格的bbox坐标呢

**Comment by Joker1212** [comment]
其实已经返回了，不仅是物理坐标，包含逻辑坐标（也就是这个box是在第几行第几列）都返回了,就在readme的quickstart里面有哈

**Comment by farierer** [comment]
好的好的，实在不好意思，没认真看，谢谢

**Comment by farierer** [comment]
您好，还是希望打扰一下，咱这个坐标是ocr取到的吗，我一直以为ocr的都是按像素点取到的整数点

**Comment by farierer** [comment]
可以取到未做平均之前的坐标吗

**Comment by Joker1212** [comment]
表格的物理坐标是直接表格模型输出的哈，ocr的坐标只是文字的坐标，表格还原的原理就是计算表格的物理框里面有哪些ocr文字的框

**Comment by Joker1212** [comment]
没有做平均吧，直接取的模型的输出坐标
